units/provisioning_user_data_done.go

```diff
@@ -134,7 +134,6 @@ func (j *userDataDoneJob) Run(ctx context.Context) {
 				TerminateIfBusy:   true,
 				TerminationReason: "failed to mount volume",
 			})
-			terminateJob.SetPriority(100)
```

Comment:
Why is this being removed?

Reply:
The Amboy job priority setting never did anything, so it has no effect (and has never had an effect). I didn't want to keep a piece of code that people may believe has an effect when it doesn't.

We could also remove it from all the other places that set it for no reason, but I don't consider it that urgent, it's just a very small piece of tech debt.

Reply:
Makes sense, it's probably worth a ticket to do eventually.

Reply:
I filed DEVPROD-3234.